From: Jan Engelhardt <jengelh@inai.de>
Date: 2023-12-26 15:16:01.213214373 +0100

wxWidgets stupidly uses $EGREP near the end of configure.in without ever setting it.

---
 configure.in |    1 +
 1 file changed, 1 insertion(+)

Index: b/configure.in
===================================================================
--- a/configure.in
+++ b/configure.in
@@ -8486,6 +8486,7 @@ dnl MAKE_SET will be replaced with "MAKE
 dnl itself (this macro is required if SUBDIRS variable is used in Makefile.am
 dnl - and we do use it)
 AC_PROG_MAKE_SET
+AC_PROG_EGREP
 
 
 AC_CONFIG_HEADERS([lib/wx/include/${TOOLCHAIN_FULLNAME}/wx/setup.h:setup.h.in])
